### PR TITLE
Fix to consider existing directives in Nginx

### PR DESCRIPTION
**Pull request by iliajie**
Dependent PR:
https://github.com/virtualmin/virtualmin-gpl/pull/349#issuecomment-1227116731

### CODE REVIEW

**Comment by jcameron** [comment]
What problem is this fixing?

**Comment by iliajie** [comment]
https://github.com/virtualmin/virtualmin-gpl/pull/436

**Comment by iliajie** [comment]
To be specific with a comment that explains it all - https://github.com/virtualmin/virtualmin-gpl/pull/436#issuecomment-1230232964.

File: virtual_feature.pl [hunk 1]
```diff
@@ -1186,16 +1186,11 @@ sub feature_get_web_php_mode
...
 if ($loc) {
 	my ($pass) = &find("fastcgi_pass", $loc);
 	if ($pass && $pass->{'words'}->[0] =~ /^(localhost|127\.0\.0\.1|unix):(.*)$/) {
-		if ($1 eq "unix" && $2 eq $fpmsock) {
-			return 'fpm';
-			}
-		elsif (($1 eq "localhost" || $1 eq "127.0.0.1") &&
-		       $fpmport && $2 eq $fpmport) {
+		if (($1 eq "localhost" || $1 eq "127.0.0.1" || $1 eq "unix") &&
+		     $2 && $2 !~ /\.sock\/socket/) {
```
Comment (jcameron) [inline virtual_feature.pl:1193]: What's special about this .sock/socket path?

Comment (iliajie) [reply virtual_feature.pl:1193]: This represents `fcgid` mode with `fcgiwrap`.

**Pull request merged**